new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -198,18 +236,22 @@ private static void initializeGenericKubernetesApiClients() {
   }
 
   // ------------------------  deployments -----------------------------------
-
   public static boolean createDeployment(String deploymentYaml) {
     // do something with the command!!!
     return true;
   }
 
-  public static List listDeployments() {
-    return new ArrayList();
+  /**
+   * List all deployments in a given namespace.
+   * @param namespace Namespace in which to list the deployments
+   * @return V1DeploymentList of deployments in the Kubernetes cluster
+   */
+  public static V1DeploymentList listDeployments(String namespace) {
+    KubernetesApiResponse<V1DeploymentList> list = deploymentClient.list(namespace);
+    return list.getObject();
```

Comment:
Shouldn't be this be checking if the call was successful?

Reply:
Fixed